### PR TITLE
Modify contrast ratios for better accessibility

### PR DESCRIPTION
**Pull request by samuel-w**
Somehow managed to break old PR: https://github.com/duplicati/duplicati/pull/4596

Maybe the coloured text could be darkened slightly as well to match the text change. To me, the green text is the one that clashes most, since the blue text is further away.

Edit: Fixes https://github.com/duplicati/duplicati/issues/4592

### CODE REVIEW

**Comment by warwickmm** [comment]
@samuel-w, are these changes the same as in pull request #4596?  If so, I would prefer to reopen that one to preserve the conversation flow.

**Comment by warwickmm** [comment]
I'm going to close this and reopen the original pull request #4596, which contains the exact same revisions.

**Comment by warwickmm** [comment]
Oops, I just noticed that the original repository for #4596 no longer exists.  Reopening this one...

**Comment by warwickmm** [comment]
Here are some screenshots with the new changes.  I think it could benefit from darkening the colored text, but that's my personal preference.  I'm not sure how it looks with the dark theme.

image

image

**Comment by warwickmm** [comment]
@samuel-w, do you see the following issue when you try to configure a new backup?

image

Can you try [building the style file](https://github.com/duplicati/duplicati/pull/4437#issuecomment-782295586) with
```
lessc -m=always -x Duplicati/Server/webroot/ngax/less/style.less > Duplicati/Server/webroot/ngax/styles/style.css
```

**Comment by samuel-w** [comment]
I think it clashes because the surrounding text and general colour scheme is so bright, especially the blue.

**Comment by samuel-w** [comment]
Changed the colours a bit. Green is a bit too contrasty.
light
dark

**Comment by samuel-w** [comment]
Is there some way to dynamically between two css files?

**Comment by samuel-w** [comment]
UI test fails even though the element is shown. Is this a test error?
test

**Comment by warwickmm** [comment]
Our UI tests sometimes have issues where the requested element isn't rendered in time.  I'll re-run the job to see if it happens again.

@ts678, @drwtsn32x, @kenkendk, any opinions on the proposed color scheme changes?

**Comment by ts678** [comment]
> any opinions on the proposed color scheme changes?

I don't use dark theme, I like increasing the contrast of small text. It's hard to get a feel for the change from screenshots, but
to me, the green-on-white after "Changed the colours a bit. Green is a bit too contrasty." [above](https://github.com/duplicati/duplicati/pull/4598#issuecomment-917784612) looks more gray then green.
On a live view of current, green and blue look similar -- kind of low contrast with the white, but they make up for it with size.

**Comment by warwickmm** [comment]
I really like the new contrast in the default theme:

image

However, some of the headings in the dark theme (Operations, Configuration, etc.) can be difficult to read:

image

For comparison, here's what the current dark theme looks like:

image

**Comment by samuel-w** [comment]
Mine looks like this:
dark
Try clearing cache or changing browser.

**Comment by warwickmm** [comment]
What does it look like if you expand the configuration (click on the down arrow)?

**Comment by samuel-w** [comment]
I completely changed how the less files work to fix the dark theme heading issue.

**Comment by samuel-w** [comment]
I completely changed how the less files work to fix the dark theme heading issue.

**Comment by samuel-w** [comment]
I completely changed how the less files work to fix the dark theme heading issue. Also darkened the text so it appears better when lightened/darkened.

**Comment by warwickmm** [comment]
If I check out revision 2d622a463dd3e88d123e4115cd84f53b371b4bc9 and build the style files, `Duplicati/Server/webroot/ngax/styles/default.css` shows up as modified.  Looking at this revision, it appears that `Duplicati/Server/webroot/ngax/styles/default.css` was committed without any changes to `Duplicati/Server/webroot/ngax/less/default.less`.  Did we forget to commit changes to the less file?

```
$ git show --stat
commit 2d622a463dd3e88d123e4115cd84f53b371b4bc9 (HEAD)
Author: samuel-w <samuel.woon@protonmail.com>
Date:   Fri Sep 24 21:45:38 2021 -0500

    Fix missed

 Duplicati/Server/webroot/ngax/less/dark.less     | 5 +++++
 Duplicati/Server/webroot/ngax/styles/dark.css    | 2 +-
 Duplicati/Server/webroot/ngax/styles/default.css | 2 +-
 3 files changed, 7 insertions(+), 2 deletions(-)
```

**Comment by samuel-w** [comment]
Yeah, I missed recompiling it. Fixed it.

**Comment by warwickmm** [comment]
Thanks.  Here's what I'm seeing now on Brave:

image
image
image
image

Looks pretty good to me.  @samuel-w, what browsers have you tested this with?  Can someone test on a few others (e.g. Edge, Chrome, Firefox, and Safari)?

**Comment by samuel-w** [comment]
I get some minor flickering when changing themes and loading when uncached. However, if we keep the old solution, then making sure all the parts of the dark theme are consistent will be a lot harder.

**Comment by warwickmm** [comment]
Thanks @samuel-w.  This looks pretty good to me.  Once we get some confirmation that things look ok in a few other browsers, I think we can merge.

File: Duplicati/Server/webroot/ngax/index.html [hunk 1]
```diff
@@ -26,8 +26,9 @@
     <meta name="theme-color" content="#ffffff">
 
     <link rel="stylesheet" type="text/css" href="styles/smoothness/jquery-ui.min.css">
-    <link rel="stylesheet" type="text/css" href="styles/style.css?v=2.0.0.7">
-    <link rel="stylesheet" type="text/css" href="styles/themes.css?v=2.0.0.7">
+    <link rel="stylesheet" type="text/css" href="styles/{{active_theme}}.css">
+	<link rel="stylesheet" type="text/css" href="styles/default.css" disabled="disabled">
+	<link rel="stylesheet" type="text/css" href="styles/dark.css" disabled="disabled">
```
Comment (warwickmm) [inline Duplicati/Server/webroot/ngax/index.html:31]: Can we use spaces for the indentation to be consistent with the surrounding code?

Comment (samuel-w) [reply Duplicati/Server/webroot/ngax/index.html]: Done.

**Comment by warwickmm** [comment]
@samuel-w, what browsers have you tested this with?

**Comment by samuel-w** [comment]
Chrome and Firefox, but I only did basic tests in Firefox

**Comment by warwickmm** [comment]
OK, I think we can merge and get some more user feedback once a new canary is released.

**Pull request merged**